documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,218 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-recovery-volume_{context}"]
+= Recovering a cluster from persistent volumes
+
+This procedure describes how to recover a deleted cluster from persistent volumes (PVs).
+
+The procedure assumes that the Topic Operator is deployed.
+In situations such as these, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+The `KafkaTopic` resources must be recovered before a cluster is started so that the topics are not deleted by the Topic Operator.
```

Comment:
This is what opetion 1 does. The option 2 does something different and doesn't recover the Kafkatopics. So maybe we should put here something generic?

Reply:
I thought a clarification of the options when you get to the step might be more useful. But this might need adjusting:

![image](https://user-images.githubusercontent.com/47596553/72725348-71895d80-3b7d-11ea-9c2a-337f1b8a6854.png)